derive_builder_macro/CHANGELOG.md

```diff
@@ -0,0 +1,225 @@
+# Change Log
```

Comment:
Is this a symlink that GitHub isn't showing properly? I don't think we'll want to maintain a dedicated changelog for the macro crate.

Reply:
It is a copy. There is a dedicated changelog for core crate, so I did a copy for macro crate also.